decentraland/unity-explorer
# fix: Unloading lod distance

## What does this PR change?

Fixes #586 

## How to test the changes?

1. Launch the explorer
2. Go to -150,150. There is a castle scene around there
3. Go flying away (F11) from the castle into the. It should eventually dissapear (Look at the behaviour in the issue video to see what shouldnt happen)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
